src/dc_receive_imf.rs

```diff
@@ -979,9 +979,14 @@ fn create_or_lookup_group(
     // XXX insert code in a different PR :)
 
     // execute group commands
-    if X_MrAddToGrp.is_some() || X_MrRemoveFromGrp.is_some() {
+    if X_MrAddToGrp.is_some() {
+        chat::reset_gossiped_timestamp(context, chat_id)?;
+        recreate_member_list = true;
```

Comment:
this is btw the precise line that causes concurrent group adds to result in inconsistent group memberships.